tool-plugins/intellij/gradle.properties

```diff
@@ -17,5 +17,5 @@
 ballerinaPluginName = ballerina-intellij-idea-plugin
 ballerinaPluginVersion = 0.980.0
 
-ideaVersion = 2016.3
+ideaVersion = 2017.3
```

Comment:
We don't need to change the version, right?

Reply:
Yeah we don't, reverted the version change